dashboard/components/DevDashboard.vue

```diff
@@ -60,15 +62,26 @@ const campaignList = computed( (): Campaign[] => Object.values( props.campaigns
 const currentCampaign = computed( (): Campaign => campaignList.value.find( ( c: Campaign ) => c.campaign === branchName.value ) );
 const filteredCampaignList = computed( (): Campaign[] => campaignList.value.filter( ( c: Campaign ) => c.campaign !== branchName.value ) );
 
-onMounted( () => {
-	fetch( '/compiled-banners/' )
-		.then( async res => {
-			if ( !res.ok ) {
-				return;
-			}
-			const fileList = await res.text();
-			compileInfo.value = parseCompileInfo( fileList );
+const campaignUrlMap = ref<Record<string, string>>( {} );
+
+onMounted( async () => {
+	try {
+		const bannersResponse = await fetch( '/compiled-banners/' );
+		if ( !bannersResponse.ok ) {
+			return;
+		}
+		const fileList = await bannersResponse.text();
+		compileInfo.value = parseCompileInfo( fileList );
+
+		const pullRequestsResponse = await fetch( `https://api.github.com/repos/wmde/fundraising-banners/pulls?state=open` );
```

Comment:
```suggestion
		const pullRequestsResponse = await fetch( `https://api.github.com/repos/wmde/fundraising-banners/pulls?state=open,closed` );
```

If we also request closed PRs we can jump to PRs that are already closed. I think that's a neat feature to have, but I'll leave that up to you.

Reply:
Done.